config/menus.js

```diff
@@ -101,10 +101,37 @@ const footer = [
   },
 ];
 
+const docs = [
+  {
+    label: "Apache Spark",
+  },
+  {
+    label: "> Batch operations",
+    url: "/docs/spark/batch",
+  },
+  {
+    label: "> Streaming operations",
+    url: "/docs/spark/streaming",
+  },
+  {
+    label: "> Update operations",
+    url: "/docs/spark/updates",
+  },
+  {
+    label: "> Utility operations",
+    url: "/docs/spark/utilities",
+  },
+  {
+    label: "> Constraints",
+    url: "/docs/spark/constraints",
+  },
+];
+
```

Comment:
Two things:

1. You can nest menu items.
2. Menu items always need a link - you can link to `/docs/spark` for the topmost link.

```suggestion
const docs = [
  {
    label: "Apache Spark",
    url: "/docs/spark",
    items: [
      {
        label: "Batch operations",
        url: "/docs/spark/batch",
      },
      {
        label: "Streaming operations",
        url: "/docs/spark/streaming",
      },
      {
        label: "Update operations",
        url: "/docs/spark/updates",
      },
      {
        label: "Utility operations",
        url: "/docs/spark/utilities",
      },
      {
        label: "Constraints",
        url: "/docs/spark/constraints",
      },
    ],
  },
];

```